config/locales/rdr.en.yml

```diff
@@ -71,58 +71,56 @@ en:
-        question14_label: "Please list the geographic area(s) where your study data was collected if significant for the analysis of the data (country, state, locality)."
-        question15_label: "Please list the dates of data collection."
-        question16_label: "Please list any languages associated with this study/dataset.  If multiple languages, please separate each language with a semicolon."
-        question17_label: "Please list the formats associated with the materials in this deposit (e.g. PDF, CSV, DICOM, WAV, TIFF). Please separate each format with a semicolon."
-        question18_label: "Please list any related publications, datasets or other published materials that you would like to associate with the study/dataset record. Enter the citation and persistent identifier (DOI)."
-        question19_label1: "Yes"
-        question19_label2: "No"
-        question20_label: "Please enter your DOI"
-        question21_label1: "I will use CC0"
-        question21_label2: "No I would not like to use CC0"
-        question22_label: "I would like to apply this creative commons license (e.g. CC BY, CC BY-NC, etc.)"
+        question9_label: "Please enter department, school, research center or other organizational affiliation.  Separate each with a semicolon."
+        question10_label: "Please list a name, phone number and/or email address for a designated contact for this study/dataset. This contact is responsible for answering questions about the data, documentation and/or code for this project."
+        question11_label: "Please enter a description for your study/dataset.  An abstract from a grant or publication is acceptable."
+        question12_label: "Please enter one or more key words to describe your study/dataset (e.g. Mechanical engineering, Sociology, Medical imaging, etc.). Please separate each key word with a semicolon."
+        question13_label: "Please list the geographic area(s) where your study data was collected if significant for the analysis of the data (country, state, locality)."
+        question14_label: "Please list the dates of data collection."
+        question15_label: "Please list any languages associated with this study/dataset.  If multiple languages, please separate each language with a semicolon."
+        question16_label: "Please list the formats associated with the materials in this deposit (e.g. PDF, CSV, DICOM, WAV, TIFF). Please separate each format with a semicolon."
+        question17_label: "Please list any related publications, datasets or other published materials that you would like to associate with the study/dataset record. Enter the citation and persistent identifier (DOI)."
+        question18_label: "please list funding agency information"
+        question19_label: "please list grant number and information"
+        question20_label1: "Yes"
```

Comment:
All the other question prompts start with uppercase letters.  Also, for the grant number one, do you really mean "grant number and information" or should it just be "grant number information"?  These prompts seem terse compared to the other ones but maybe that's what the product owners want?